stable/prometheus-pushgateway/templates/_helpers.tpl

```diff
@@ -52,3 +52,26 @@ Create default labels
 {{- $labels := dict "app" $labelApp "chart" $labelChart "release" .Release.Name "heritage" .Release.Service -}}
 {{ merge .extraLabels $labels | toYaml | indent 4 }}
 {{- end -}}
+
+{{/*
+Create unified labels for prometheus components
+*/}}
+{{- define "prometheus-pushgateway.common.matchLabels" -}}
+app: {{ template "prometheus-pushgateway.name" . }}
+release: {{ .Release.Name }}
+{{- end -}}
+
+{{- define "prometheus-pushgateway.labels" -}}
+{{ include "prometheus-pushgateway.common.matchLabels" . }}
+{{ include "prometheus-pushgateway.common.metaLabels" . }}
+{{- end -}}
+
+{{- define "prometheus-pushgateway.matchLabels" -}}
+component: {{ .Values.name | quote }}
+{{ include "prometheus-pushgateway.common.matchLabels" . }}
+{{- end -}}
+
+{{- define "prometheus-pushgateway.common.metaLabels" -}}
+chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+heritage: {{ .Release.Service }}
+{{- end -}}
```

Comment:
@paprickar it seems the changes include more than just "Add persistence support". Pls open a seperate PR for different   functionality

Reply:
@cstaud thanks for pointing this out.
I updated my PR and adapted to the current label handling. 
Let me know if this is correct.